typings/index.d.ts

```diff
@@ -266,31 +266,39 @@ interface ArrayGraphImpl<NativeType extends any[], F extends FunctionLibrary,
 
     /**
      * Joins an array of strings to a single strings, like NativeType.join(separator)
-     * @param separator 
+     * @param separator
      */
     join(separator: Argument<string>): Value extends string ? StringGraph<string, F> : never
 
     reverse(): ArrayGraph<Value[], F>
 
     /**
      * Runs the functor for every item in an array. Returns a graph that resolves to an array with the returned values.
-     * 
+     *
      * @param functor A function to run for every item of the array
      * @param scope A variable to pass to the functor if inside another functor.
      */
     map<Scope, Ret>(functor: (value: ValueGraph, key?: KeyGraph, scope?: Scope) => Argument<Ret>, scope?: Scope) : ArrayGraph<Ret[], F>
 
     /**
      * Returns a boolean graph that resolves to true if running the functor on any of the array's item resolved to true
-     * 
+     *
      * @param functor A function to run for every item of the array, returning boolean
      * @param scope A variable to pass to the functor if inside another functor.
      */
     any<Scope>(functor: (value: ValueGraph, key?: KeyGraph, scope?: Scope) => Argument<boolean>, scope?: Scope) : BoolGraph<F>
 
+    /**
+     * Returns a boolean graph that resolves to true if running the functor on all of the array's items resolved to true
+     *
+     * @param functor A function to run for every item of the array, returning boolean
+     * @param scope A variable to pass to the functor if inside another functor.
+     */
+    every<Scope>(functor: (value: ValueGraph, key?: KeyGraph, scope?: Scope) => Argument<boolean>, scope?: Scope) : BoolGraph<F>
```

Comment:
@noamr please excuse my editor removing all the extra spaces at the ends of the lines, this is the actual change.